### PR TITLE
Cotton/Durathreads now stack to 80

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## About The Pull Request

Makes storge and handling of large amouts of cotton/durathreads much easyer

## Why It's Good For The Game

Hatter suggested it thus it might be good!
Also allows for better storge of the plentiful amouts of mass made cottons 
## Changelog
:cl:
tweak: Cotton/Durathread now stack to 80 bundles 
/:cl:

### CODE REVIEW

**Comment by Ragolution** [comment]
But why not 100?

**Comment by Trilbyspaceclone** [comment]
@Ragolution Dont want them to become to bit + spam click abuse...

**Comment by davidchan13** [comment]
Does this also affect silk?

File: code/game/objects/items/stacks/sheets/sheet_types.dm [hunk 1]
```diff
@@ -809,6 +810,7 @@ new /datum/stack_recipe("paper frame door", /obj/structure/mineral_door/paperfra
 /obj/item/stack/sheet/cotton/durathread
 	name = "raw durathread bundle"
 	desc = "A bundle of raw durathread ready to be spun on the loom."
+	max_amount = 80
```
Comment (ghost) [inline code/game/objects/items/stacks/sheets/sheet_types.dm]: no in-game impact but durathread doesn't need the max_amount variable, it inherits max_amount = 80 from /obj/item/stack/sheet/cotton

Comment (Trilbyspaceclone) [reply code/game/objects/items/stacks/sheets/sheet_types.dm]: Aware its their for clarity. Like how some times you see extra vars/defines on child versos of other mats.

**Comment by Trilbyspaceclone** [comment]
@davidchan13 Nope

File: code/game/objects/items/stacks/sheets/sheet_types.dm (hunk 1 above)
Comment (Ghommie) [reply code/game/objects/items/stacks/sheets/sheet_types.dm]: That's not a good reason though, considering items, stacks and sheets have dozens and dozens variables and restating them is a waste of bytes.

**Review by Ghommie — APPROVED** [review]
I really don't understand why but ok.

**Pull request merged**